controllers/endPoints.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchEndPoints } = require('../models/endPoints.model');
+
+exports.getEndPoints = (request, response, next) => {
+    fetchEndPoints()
+    .then(endpoint => {
+    response.status(200).send(endpoint);
```

Comment:
Remember: in order to be RESTful (and readable) - all of our responses should be sent on an object with a key of what they contain. If you put your response on a key that describes what it is - what would the best key be? Something like {something: endpoints}